omegaconf/grammar_visitor.py

```diff
@@ -229,6 +219,13 @@ def visitListContainer(
     def visitPrimitive(self, ctx: OmegaConfGrammarParser.PrimitiveContext) -> Any:
         return self._createPrimitive(ctx)
 
+    def visitQuotedValue(self, ctx: OmegaConfGrammarParser.QuotedValueContext) -> str:
+        # (QUOTE_OPEN_SINGLE | QUOTE_OPEN_DOUBLE)
+        #     (interpolation | ESC | ESC_INTER | SPECIAL_CHAR | ANY_STR)*
+        # MATCHING_QUOTE_CLOSE;
```

Comment:
maintaining those comments looks like a pain in the ass.
With the grammar stabilizing I don't think it's a big deal, but I don't think I would bother to do it.

Reply:
It's a trade-off -- some maintenance overhead vs. easier-to-understand code. I have my own preference but I don't mind removing the comments either, it's not a big deal.

Reply:
No need to remove yet. If they start to fall out of sync in the future I will reconsider.